packages/xo-server-load-balancer/src/plan.js

```diff
@@ -314,9 +314,19 @@ export default class Plan {
   async _processVcpuPrepositioning(hosts) {
     const promises = []
 
-    const idToHost = keyBy(hosts, 'id')
+    // removing hosts which have incorrect cpu count value to avoid mass migration on rrd malfunction
+    const sanitizedHostList = hosts.filter(host => parseInt(host.CPUs.cpu_count) > 0)
+    if (sanitizedHostList.length < hosts.length) {
+      const unhealthyHost = hosts.find(host => !(parseInt(host.CPUs.cpu_count) > 0))
```

Comment:
`hosts.CPUs` is deprecated. Use `host.cpus` instead (`host.cpus.cores` it can be undefined BTW)
https://github.com/vatesfr/xen-orchestra/blob/master/%40vates/types/src/xo.mts#L172